.stickler.yml

```diff
@@ -0,0 +1,15 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: "./.eslintrc.json"
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+    - "dist/*"
+    - "node_modules/*"
```

Comment:
Node modules can be safely removed from here since it's in .gitigonre and therefore not being pushed in the repo.